makefiles/suit.v3.base.inc.mk

```diff
@@ -0,0 +1,40 @@
+#
+# path to suit-tool
+SUIT_TOOL ?= python $(RIOTBASE)/dist/tools/suit_v3/suit-manifest-generator/bin/suit-tool
```

Comment:
this probably needs to be "python3"

Reply:
I've modified it to rely on the hashbang in `suit-tool`